packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/lib/base/data_sources/local/auth_token_secure_data_source.dart

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) 2021, Prime Holding JSC
+// https://www.primeholding.com
+//
+// Use of this source code is governed by an MIT-style
+// license that can be found in the LICENSE file or at
+// https://opensource.org/licenses/MIT.
+
+import 'package:flutter_secure_storage/flutter_secure_storage.dart';
+
+import 'auth_token_data_source.dart';
+
+/// Concrete implementation of AuthTokenDataSource using FlutterSecureStorage.
+/// Suitable for mobile.
+/// Persist and get auth information to make it  available trough the app.
+class AuthTokenSecureDataSource implements AuthTokenDataSource {
+  AuthTokenSecureDataSource(){
+    _storage = const FlutterSecureStorage();
```

Comment:
Please use explicit dependencies by injecting them/it in the constructor. To make it easier to be used you can use default parameters